src/Rules/RuleRunner.cs

```diff
@@ -87,7 +87,12 @@ private static RunResult RunRuleWorker(IRule rule, IA11yElement element)
             if (!ShouldRun(rule.Condition, element))
                 return result;
 
-            result.EvaluationCode = rule.Evaluate(element);
+            // The PassesTest call is the method we want to convert all rules to use
+            // Using RuleInfo.ErrorCode will allow us to use reflection to document the result of each rule
+            if (rule.Info.ErrorCode.HasValue)
```

Comment:
Should there be a try/catch around these, so that we can accommodate the rules that currently return ErrorCode.RuleExecutionError? I was thinking that the catch block would set result.EvaluationCode to RuleExecutionError...